### PR TITLE
Add more visually pleasing logging

### PR DESCRIPTION
**Pull request by Albert221**
image
image

### CODE REVIEW

**Comment by bartekpacia** [comment]
Just in case you didn't know about it: https://github.com/schollz/progressbar

File: log/logger.go [hunk 1]
```diff
@@ -0,0 +1,54 @@
+// Package log provides beautiful logger for the console interface.
+package log
+
+import (
+	"fmt"
+	"io"
+	"strings"
+
+	clr "github.com/TwiN/go-color"
```
Comment (bartekpacia) [inline log/logger.go:9]: Does it automatically disable colors on older (non-ANSI) terminals?

File: log/logger_test.go [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+	"github.com/leancodepl/poe2arb/log"
+	"github.com/stretchr/testify/assert"
+)
+
+const (
+	blue  = "\x1b[34m"
+	green = "\x1b[32m"
+	red   = "\x1b[31m"
+	reset = "\x1b[0m"
+)
```
Comment (bartekpacia) [inline log/logger_test.go:16]: ah, okay

**Review by bartekpacia — APPROVED** [review]

File: log/logger.go (hunk 1 above)
Comment (Albert221) [reply log/logger.go:9]: No, but I'm wondering whether any practical use-case (developer's terminal or CI) is non-ANSI

Comment (bartekpacia) [reply log/logger.go:9]: Yeah, probably not. I'm fine if you ignore it.

**Pull request merged**